src/helpers/error-message.helper.ts

```diff
@@ -146,12 +146,28 @@ export class ErrorMessageHelper {
         title = `${error.worker}`;
         break;
       case AlgorithmErrorCode.WorkerUnderTime:
-        message = `Pracownik ma <strong>${error.hours}</strong> niedogodzin.`;
+        message = `Pracownik ma <strong>${error.hours}</strong> niedogodzin`;
+        if (error.hours === 1) message += `ę`;
+        if (
+          (error.hours - 2) % 10 === 0 ||
+          (error.hours - 3) % 10 === 0 ||
```

Comment:
Piękne to jest <3 
Z tym, że znalazłem lukę dla "nastu"
Powinno być:
Pracownik XYZ ma 13 nadgodzin
A będzie:
Pracownik XYZ ma 13 nadgodziny

Reply:
niedogodzin* 
Btw @Qwebeck podpowiada, że można by to wynieść do funkcji:
generateInflectionalEnding :D